sdk/cosmosdb/cosmos/src/utils/batch.ts

```diff
@@ -29,6 +29,10 @@ export interface OperationResponse {
   resourceBody?: JSONObject;
 }
 
+export interface BulkOptions {
+  continueOnError?: boolean;
```

Comment:
Add some docs about what this does and make clear this defaults to false

Reply:
Also add a note to saying we plan to change this to true in the next major release. Add a note to the v4 thread